v3/auth.go

```diff
@@ -10,9 +10,19 @@ import (
 // Renew login 10 minutes before we have to
 const renewBefore int64 = 600
 
-func (e *Exchange) Authenticate(key, secret string) error {
+// Keep clientID and clientSecret for re-connecting automatically
+var (
+	clientID     string
+	clientSecret string
+)
+
+func (e *Exchange) Authenticate(keys ...string) error {
```

Comment:
Can you add a docblock to explain what arguments need to be passed here please? Just so users do not need to read the code to understand how to use the method.